### PR TITLE
build(deps-dev): replace standard with neostandard

### PR DESCRIPTION
**Pull request by Fdawgs**
See https://github.com/fastify/fastify-cors/pull/340. This is a batch PR, so may have some issues. Please review changes.

### CODE REVIEW

**Comment by arthurfiorette** [comment]
Sure! let me take a look at it

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "@fastify/pre-commit": "^2.1.0",
     "@types/node": "^22.0.0",
     "fastify": "^5.0.0",
```
Comment (Fdawgs) [inline package.json:34]:
```suggestion
    "eslint": "^9.17.0",
    "fastify": "^5.0.0",
```

File: lib/httpError.d.ts [hunk 2]
```diff
@@ -92,13 +93,13 @@ export type HttpErrorNames = 'badRequest'
...
+                    | 'networkAuthenticationRequired'
 
 export type HttpErrors = {
   HttpError: typeof HttpError;
   getHttpError: (code: HttpErrorCodes, message?: string) => HttpError;
   createError: (...args: UnknownError[]) => HttpError;
-} & Record<HttpErrorNames, (msg?: string) => HttpError>;
+} & Record<HttpErrorNames, (msg?: string) => HttpError>
 
 declare const HttpErrors: HttpErrors
```
Comment (arthurfiorette) [inline lib/httpError.d.ts:104]:
```suggestion
declare const httpErrors: HttpErrors
```

File: lib/httpError.d.ts [hunk 3]
```diff
@@ -92,13 +93,13 @@ export type HttpErrorNames = 'badRequest'
...
-export default HttpErrors;
+export default HttpErrors
```
Comment (arthurfiorette) [inline lib/httpError.d.ts:105]:
```suggestion
export default httpErrors
```

**Review by arthurfiorette — CHANGES_REQUESTED** [review]

File: lib/httpError.d.ts (hunk 3 above)
Comment (arthurfiorette) [inline lib/httpError.d.ts:105]:
```suggestion
export default httpErrors
```

File: lib/httpError.d.ts (hunk 2 above)
Comment (arthurfiorette) [inline lib/httpError.d.ts:104]:
```suggestion
declare const HttpErrors: httpErrors
```

**Review by arthurfiorette — CHANGES_REQUESTED** [review]

**Pull request merged**